formbricks/formbricks
# feat: Create new team

## What does this PR do?

- Allow users to delete their only team
- Redirect user to `/create-first-team` if they have no team

Fixes #1816 


Loom Video: https://www.loom.com/share/843147c04785426eb667b5a2c664bc00?sid=e1a9b526-fa43-4536-b446-4c6d10bd9784

## How should this be tested?



- Login
- Delete all teams (You should be able to delete the last team )
- You should get redirected to `create-first-team`
- Create a team
- You should get redirected to `environments/environmentId/surveys`

## Checklist



### Required


### Appreciated
